WiseUnpacker/SabreTools.Models.WiseInstaller/SectionHeader.cs

```diff
@@ -136,10 +136,44 @@ public class SectionHeader
         /// Byte array representing version. Byte array used due to unknown size and type for version.
         /// </summary>
         public byte[]? Version { get; set; }
+        
+        /// <summary>
+        /// String representing the WIS[etc].TMP string
+        /// </summary>
+        public string? TmpString { get; set; }
+        
+        /// <summary>
+        /// String representing the GUID string.
+        /// </summary>
+        public string? GuidString { get; set; }
+        
+        /// <summary>
+        /// String representing a version number. This isn't the version of the .WISE installer itself, as it is
+        /// entirely inconsistent even within the same week. Likely refers to a version for what's being installed
+        /// rather than the installer itself
+        /// </summary>
+        public string? NonWiseVersion { get; set; }
```

Comment:
If I recall correctly, this is actually after the strings at the end. Please put it there. The models should be "in order" when possible. Interleaved values are an exception to this.

Reply:
This isn't after the strings; this is in between the GUID and the pre-string byte values. The model is already in order